test/test_integration.rb

```diff
@@ -69,16 +69,8 @@ def wait_booted
   def restart_server(connection)
     signal :USR2
 
-    sleep 1
-
     connection.write "GET / HTTP/1.1\r\n\r\n" # trigger it to start by sending a new request
 
-    assert_raises Errno::ECONNRESET do
```

Comment:
I guess it's testing that the server *should not* respond within 2 seconds? That's...not really testing whether or not the server restarted lol.